src/styles/calendar.less

```diff
@@ -0,0 +1,146 @@
+// -------- CALENDAR COMPONENT, MODAL STYLED AT BOTTOM --------
+
+.ant-btn {
+  background-color: #fcdfaa !important;
+  position: absolute !important;
+  right: 5%;
+  top: 7.8%;
+  width: fit-content;
+  padding: 0 5% !important;
+  color: #f27712 !important;
+  font-weight: bold !important;
+}
+
+.ant-picker-calendar-header {
+  justify-content: flex-start !important;
+  margin: 0 2% !important;
+  div {
+    background-color: #f1c461 !important;
+    color: #f27712 !important;
+  }
+}
+
+.ant-radio-button-wrapper {
+  background-color: #fcdfaa !important;
+  color: #f27712 !important;
+}
+.ant-radio-button-wrapper-checked {
+  background-color: #f1c461 !important;
+  color: white !important;
+}
+
+thead {
+  background-color: #fcdfaa;
+}
+
+th {
+  padding: 2rem !important;
+}
+
+thead tr th {
+  color: #f27712 !important;
+}
+
+ul {
+  background-color: transparent !important;
+}
+
+.ant-badge-not-a-wrapper {
+  text-align: center;
+  margin-top: 5% !important;
+  padding: 4px 16px !important;
+  width: 100% !important;
+  .ant-badge-status-text {
+    color: black !important;
+    margin-left: 0 !important;
+  }
+}
+.ant-badge-not-a-wrapper:nth-of-type(1) {
+  margin-top: 0 !important;
+}
+
+.success-class-event,
+.warning-class-event,
+.error-class-event {
+  border-radius: 9999px !important;
+}
+
+.success-class-event {
+  background-color: rgba(0, 202, 183, 0.25) !important;
+}
+
+.warning-class-event {
+  background-color: #faad14 !important;
+}
+
+.error-class-event {
+  background-color: salmon !important;
+}
+
+.ant-badge-status-dot {
+  display: none !important;
+}
+
+.ant-picker-calendar {
+  overflow: hidden !important;
+  padding: 2% !important;
+}
+
+.ant-picker-panel {
+  border: 2px solid #fbecd1 !important;
+  border-radius: 5rem !important;
+  overflow: hidden !important;
+}
+
+.ant-picker-body {
+  padding: 0 !important;
+}
+
+td.ant-picker-cell {
+  border: 2px solid #fbecd1 !important;
+}
+
+.ant-picker-cell-inner {
+  border-top: none !important;
+  padding: none !important;
+  margin: 0 !important;
+  background-color: #faf9f8 !important;
+  .ant-picker-calendar-date:hover {
+    background-color: #096a70 !important;
+    color: white !important;
+  }
+}
+
+.ant-picker-calendar-date:hover {
+  background-color: #096a70 !important;
+  div,
+  ul span {
+    color: white !important;
+  }
+}
+
+tr:nth-of-type(6) {
+  td {
+    border-bottom: none !important;
+  }
+}
+
+div.ant-picker-calendar-date-value {
+  color: #f27712 !important;
+}
+
+// ---------- MODAL ----------
+.ant-picker-date-panel {
```

Comment:
I like the idea of the modal being used when adding events to the calendar. It separates this from looking at the calendar. Good job overall on the design of the calendar.